docs/flutter/v2/features/background-handling.mdx

```diff
@@ -148,65 +148,114 @@ Add service inside `<application>` tag as follows:
-      androidNotificationOptions: AndroidNotificationOptions(
-        channelId: 'notification_channel_id',
-        channelName: 'Foreground Notification',
-        channelDescription: 'This notification appears when the foreground service is running.',
-        channelImportance: NotificationChannelImportance.LOW,
-        priority: NotificationPriority.LOW,
-        iconData: NotificationIconData(
-          resType: ResourceType.mipmap,
-          resPrefix: ResourcePrefix.ic,
-          name: 'launcher',
-        ),
-      ),
-      iosNotificationOptions: const IOSNotificationOptions(
-        showNotification: false,
-        playSound: false,
-      ),
-      foregroundTaskOptions: const ForegroundTaskOptions(
-        interval: 5000,
-        autoRunOnBoot: false,
-        allowWifiLock: false,
-      ),
-      notificationTitle: 'Foreground Service is running',
-      notificationText: 'Tap to return to the app',
-      callback: startCallback,
-      child: Scaffold(
-        appBar: AppBar(
-          title: const Text('Your App Title'),
-          centerTitle: true,
-        ),
-        body: MeetingApp(), // your app screen which is shown when 100ms Room is joined
-      ),
-    ),
+    /// A widget that prevents the app from closing when the foreground service is running.
+    /// This widget must be declared above the [Scaffold] widget.
+    home: WithForegroundTask(
+      child: MeetingPage()
+    );
   );
 }
 ```
 </div>
 
+### Start Foreground Service
+
+Start `FlutterForegroundTask` at `onJoin` callback.
+
+```dart
+@override
+  void onJoin({required HMSRoom room}) async {
+    // Start FlutterForegroundTask at onJoin callback.
+      FlutterForegroundTask.startService(
+        notificationTitle: "100ms foreground service running",
+        notificationText: "Tap to return to the app");
+      ...
+  }
+```
+
+### Stop Foreground Service
+
+Stop `FlutterForegroundTask` at `onRemovedFromRoom` and `onSuccess` -> `HMSActionResultListenerMethod.leave` callbacks.
+
+```dart
+
+class Meeting implements HMSActionResultListener{
+  
+  void leave(){
+    //this is the instance of class implementing HMSActionResultListener  
+    await hmsSDK.leave(hmsActionResultListener: this);
+  }
+
+  @override
+  void onRemovedFromRoom({required HMSPeerRemovedFromPeer hmsPeerRemovedFromPeer}) {
+    // Clear the local room state
+    // Stop FlutterForegroundTask
+    FlutterForegroundTask.stopService();
+    ...
+  }
+
+  @override
+  void onSuccess({HMSActionResultListenerMethod methodType = HMSActionResultListenerMethod.unknown, Map<String, dynamic>? arguments}) {
+    switch (methodType) {
+      case HMSActionResultListenerMethod.leave:
+      // Room leaved successfully
+      // Clear the local room state
+      // Stop FlutterForegroundTask
+        FlutterForegroundTask.stopService();    
+        break;
+      ...
+    }
+  }
+```
+
 That's it. Now you'll have a Foreground Service up & running in your app.
 
+
+<video controls>
```

Comment:
Add a section header to the video & mention about it Overview section of the Doc